src/components/Header/index.tsx

```diff
@@ -118,6 +117,18 @@ const HeaderLinks = styled(Row)`
 `};
 `
 
+const ComingSoonBadge = styled.div`
```

Comment:
This badge is used in the token picker too (I was thinking about making it its own reusable component). Could you do it right now so that it's already usable for me and anyone else who needs it please?

Reply:
@luzzif we can optimize it later, lets merge code and create more components later.

Reply:
I guess at least an added `TODO` would help us keeping track of the thing and be enough for now? Otherwise the risk is to forget about it and end up with a mixed approach (some of the app uses shared components and some uses locally-defined, possibly code-duplicated components).